v2/src/theme/NavbarItem/RecipeLabelItem.js

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
-import { isMobile } from 'react-device-detect';
+import { BrowserView, isMobile } from 'react-device-detect';
```

Comment:
https://user-images.githubusercontent.com/18233774/188368841-cf8f3905-ae93-4e8d-b568-6ca63696dbd5.MP4

Still seeing the issue (Video from iOS safari)

Reply:
so its a SSG case. now its fixed by using docusaurus's `BrowserOnly`